src/main/java/xyz/vegaone/easytrackingv3/controller/ProjectController.java

```diff
@@ -0,0 +1,64 @@
+package xyz.vegaone.easytrackingv3.controller;
+
+import lombok.NonNull;
+import org.springframework.web.bind.annotation.*;
+import xyz.vegaone.easytrackingv3.dto.Project;
+import xyz.vegaone.easytrackingv3.dto.Sprint;
+import xyz.vegaone.easytrackingv3.dto.Task;
+import xyz.vegaone.easytrackingv3.dto.User;
+import xyz.vegaone.easytrackingv3.service.ProjectService;
+
+import java.util.List;
+
+import static xyz.vegaone.easytrackingv3.controller.RestApiConstants.BASE_URL;
+
+@RestController
+@RequestMapping(BASE_URL+"project")
+public class ProjectController {
+
+    private ProjectService projectService;
+
+    public ProjectController(ProjectService projectService) {
+        this.projectService = projectService;
+    }
+
+    @PostMapping
+    public Project createProject(@NonNull @RequestBody Project project){
+        return projectService.createProject(project);
+    }
+
+    @PutMapping
+    public Project updateProject(@NonNull @RequestBody Project project){
+        return projectService.updateProject(project);
+    }
+
+    @GetMapping("{id}")
+    public Project getProject(@PathVariable Long id){
+        return projectService.getProject(id);
+    }
+
+    @GetMapping("all")
+    public List<Project> getAllProject(){
+        return projectService.getAllProjects();
+    }
+
+    @DeleteMapping("{id}")
+    public void deleteProject(@PathVariable Long id){
+        projectService.deleteProject(id);
+    }
+
+    @GetMapping("/sprint/{projectId}")
```

Comment:
We need a different path pattern, though I'm not sure which one to use. I will research a bit.
As it is now it might be confusing. e.g. `/api/v1/project/sprint/1234` might be interpreted as getting the sprint with id 1234.

Later edit: I think I found some examples [here](https://docs.microsoft.com/en-us/graph/api/group-getmembergroups?view=graph-rest-1.0&tabs=http). 

How do you feel if we switch to something like `/{projectId}/sprint`? A request will look something like `/api/v1/project/1234/sprint`